galadriel/tools/web3/onchain/solana/spl_token.py

```diff
@@ -90,11 +91,105 @@ def forward(self, user_address: str, token_address: str) -> Optional[float]:
             return None
 
 
+class TransferTokenTool(SolanaBaseTool):
+    """Tool for transferring SPL tokens between user
+
+    Attributes:
+        name (str): Tool identifier
+        description (str): Description of the tool's functionality
+        inputs (dict): Schema for required input parameters
+        output_type (str): Type of data returned by the tool
+    """
+
+    name = "transfer_token"
+    description = "Transfers SPL tokens between users."
+    inputs = {
+        "recipient_address": {
+            "type": "string",
+            "description": "The address of the recipient.",
+        },
+        "token_address": {
+            "type": "string",
+            "description": "The SPL token mint address.",
+        },
+        "amount": {
+            "type": "number",
+            "description": "The amount of tokens to transfer.",
```

Comment:
It would make sense to specify token's precision i.e if amount is 1 am I sending one token or 1*10^(-18) tokens?